### PR TITLE
Merged pull request...

### PR DESCRIPTION
**Pull request by jedie**
It is time-consuming to maintain all changes in separate pull requests.

So i merged all changes from #9 #10 #11 and #13

### CODE REVIEW

File: tests/myapp/models.py [hunk 1]
```diff
@@ -4,7 +4,44 @@
 from publisher.models import PublisherModel
 
 
+try:
+    import parler
+except ImportError:
+    PARLER_INSTALLED=False
```
Comment (suutari-ai) [inline tests/myapp/models.py]: Just set `parler = None` and use `if parler:` to check if it's available.

File: tests/myapp/models.py [hunk 2]
```diff
@@ -4,7 +4,44 @@
...
+else:
+    PARLER_INSTALLED = True
+
+
+try:
+    import aldryn_translation_tools
+except ImportError as err:
+    TRANSLATION_TOOLS_INSTALLED=False
```
Comment (suutari-ai) [inline tests/myapp/models.py]: same here, use `aldryn_translation_tools = None` for the except part. Also, remove the `as err`.

Comment (jedie) [reply tests/myapp/models.py]: done in https://github.com/andersinno/django-model-publisher-ai/pull/14/commits/0379bcb3525089541fd33d8bb9510e0bf4f734e5

**Comment by jedie** [comment]
Anything else that needs to be changed to merged?

**Comment by suutari-ai** [comment]
Sorry, but this PR is too big. My time for this project is quite limited and reviewing all these unrelated changes in a single PR needs bigger "timeslot" than I have available for a while for this.

The start_date/end_date thing started to look ready for merging before you combined it here, so maybe just revive it in the separate PR?

**Comment by jedie** [comment]
That's a dilemma. My time is also limited. That's why I merged the changes.

And there's also another new thing coming up: I need a role "request publishing" and "accept/reject publish request" with short message exchange.

All this is too time-consuming to implement completely separately. Maybe i should made my fork independent and publish a new package to PyPi?!?

**Comment by JsseL** [comment]
@jedie I think there is value in focusing development into one repo. So if you want, we can still try to help you out in ironing the features. Just that, the smaller and more concise the PR, the easier it is to review and iterate on.

**Comment by jedie** [comment]
> I think there is value in focusing development into one repo. 

I agree with you. But the reactions times are very long or there is no response. e.g.:

#11 -> a simple bugfix + editorconfig, 8 days old -> No reaction
#10 8 days old, has some questions -> No reaction

Or the hint at #5 ... no reaction...

I think the interest in this project is very low.

**Comment by JsseL** [comment]
@jedie You are right, this is not actively maintained project, rather a occasionally maintained fork from an abandoned project.

So indeed, it might make sense to have your own fork to keep things going forward in your projects.

**Comment by jedie** [comment]
I have made a fork, merge all my changes and release this on PyPi as **django-ya-model-publisher**:

github: https://github.com/wearehoods/django-ya-model-publisher/
PyPi: https://pypi.org/project/django-ya-model-publisher/

**Pull request closed without merging**